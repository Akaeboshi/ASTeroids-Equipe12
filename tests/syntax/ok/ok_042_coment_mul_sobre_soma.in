/* mul over add */ 2  +   3   * 4;
