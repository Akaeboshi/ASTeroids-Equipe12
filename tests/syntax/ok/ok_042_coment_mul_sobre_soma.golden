Block
  Expression
    Binary(+)
      Int(2)
      Binary(*)
        Int(3)
        Int(4)
